bindings/nodejs/examples/exchange/3-check-balance.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Wallet } from '@iota/sdk';
+
+// This example uses secrets in environment variables for simplicity which should not be done in production.
+require('dotenv').config({ path: '.env' });
+
+// Run with command:
+// yarn run-example ./exchange/3-check-balance.ts
+
+// This example gets the balance of an account.
+async function run() {
+    try {
+        if (!process.env.WALLET_DB_PATH) {
+            throw new Error(
+                '.env WALLET_DB_PATH is undefined, see .env.example',
+            );
+        }
+
+        const wallet = new Wallet({
+            storagePath: process.env.WALLET_DB_PATH,
+        });
+
+        const account = await wallet.getAccount('Alice');
+        const addresses = await account.addresses();
+
+        console.log('Addresses:', addresses);
+
+        // Set syncOnlyMostBasicOutputs to true if not interested in outputs that are timelocked,
+        // have a storage deposit return, expiration or are nft/alias/foundry outputs.
+        const balance = await account.sync({ syncOnlyMostBasicOutputs: true });
```

Comment:
Maybe good to also call `account.setDefaultSyncOptions({ syncOnlyMostBasicOutputs: true })` in case someone copies code from other examples where nothing is provided

Reply:
Done